frontend/src/metabase/modes/components/drill/DashboardClickDrill.tsx

```diff
@@ -108,7 +116,10 @@ export default ({ question, clicked }) => {
         clickBehavior,
       });
 
-      const path = Urls.dataAppPage({ id: dataAppId }, { id: targetId });
+      const path = Urls.dataAppPage(
+        { id: dataAppId } as any,
+        { id: targetId } as any,
+      );
```

Comment:
Please don't disable type checking

Reply:
Oh, I now see it didn't make sense from types perspective before that, my bad. In any case, we should really avoid adding `as any` all around here

Reply:
My original solution was to change `dataAppPage` argument annotations to `Pick<..., "id">` which keeps information in the annotation without restricting the type so much, but I changed to the `any` because of a conflict with some data app changes.

What do you think about using `Pick`?

Reply:
`Pick` sounds great to me. The reason we're passing full objects is that we often need a `name` property to build a slug as well, but there are many cases when we need a URL, and we don't have the full object that'd suit TypeScript. `Pick` sounds as a way to go here 👍